MyHeartCounts/Resources/Localizable.xcstrings

```diff
@@ -4796,6 +4955,12 @@
           }
         }
       }
+    },
+    "Workout Preference" : {
+
+    },
+    "WORKOUT_PREFERENCE_TEXT" : {
```

Comment:
We can use the following text to get started:
"The MyHeart Counts app will send you personalized notifications to help you stay active. Tell us which workouts you prefer and when during the day you’d like to receive reminders. This helps us tailor your experience to your schedule and goals."